analyses/fusion-frequencies/01-fusion-frequencies.R

```diff
@@ -196,7 +196,6 @@ fusion_df <- fusion_df  %>%
   # replace NA to "" in columns that have NA
   replace_na(list("Kinase_domain_retained_Gene1A"="",
                   "Kinase_domain_retained_Gene1B"="",
-                  "Reciprocal_exists_either_gene_kinase"="",
```

Comment:
@jharenza I reviewed the data in that column as I ran the module with v11 data, and that column does not have any 'NA'. That's why I removed it. The `replace_na` function only takes a list of column names where NA is found. Unless I remove `Reciprocal_exists_either_gene_kinase`, that chunk will throw an error.

Reply:
Hmm, I think there should be NAs for all fusions which do not contain kinases as well as the DGD data because that information is unknown. For general code maintenance, we should try not to remove lines such as this, but instead troubleshoot - why were there NAs in v10 but not v11, something must not be right because the majority of v11 data is v10 data. Can you look into this please?

Reply:
@jharenza  I reviewed the script lines 193-199 to reproduce `'NA'` for `Reciprocal_exists_either_gene_kinase`. But when I ran the module, both using v10 and v11 data, there were no `NA`s found for that column.